nilearn/signal.py

```diff
@@ -71,7 +72,9 @@ def _standardize(signals, detrend=False, standardize='zscore'):
                 signals = signals - signals.mean(axis=0)
 
             std = signals.std(axis=0)
-            std[std < np.finfo(np.float64).eps] = 1.  # avoid numerical problems
+            std[
+                std < np.finfo(np.float64).eps
+            ] = 1.0  # avoid numerical problems
```

Comment:
I find this less readable, but maybe it's a matter of taste. Was the line too long ?